### ISSUE TITLE
Unions cannot be used with issubclass()

### ISSUE THREAD

**Issue by blyxxyz** [The-Compiler/qutebrowser#2090]
The Python 3 version currently in Debian unstable (`3.5.2+`) rejects this in line 430 of `commands/command.py`:
```python
        elif issubclass(typ, typing.Union):
            # this is... slightly evil, I know
```
It gives a `TypeError` of `Unions cannot be used with issubclass().` because `typing.Union` can't be subclassed or instantiated.

The only working substitute I've found is using `isinstance` with `typing._Union`, but that seems like a bad solution.

**Comment by The-Compiler** [comment]
Ouch... I think `type(typ) is typing.Union` would probably work?

**Comment by blyxxyz** [comment]
No, and `type(typing.Union[str, int]) == typing.Union` gives a `RecursionError`.

**Comment by The-Compiler** [comment]
Fixed in b6195d4e77e1309b4b4ecf77508a9ad65b007fcc and released v0.8.3 with the fix.
